go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -697,6 +698,61 @@ var _ = Describe("User Defined Network Controller", func() {
 					}
 				})
 			})
+
+			It("when started, CR exist, stale NADs exist, should deleted stale NADs", func() {
+				var testObjs []runtime.Object
+				staleNADsNsNames := []string{"red", "blue"}
+				staleLabel := map[string]string{"test.io": "stale"}
+				for _, nsName := range staleNADsNsNames {
+					ns := testNamespace(nsName)
+					ns.SetLabels(staleLabel)
+					testObjs = append(testObjs, ns)
+				}
+				connectedNsNames := []string{"green", "yellow"}
+				connectedLabel := map[string]string{"test.io": "connected"}
+				for _, nsName := range connectedNsNames {
+					ns := testNamespace(nsName)
+					ns.SetLabels(connectedLabel)
+					testObjs = append(testObjs, ns)
+				}
+				rand.Shuffle(len(testObjs), func(i, j int) {
+					testObjs[i], testObjs[j] = testObjs[j], testObjs[i]
+				})
```

Comment:
interesting, what do we achieve with that?

Reply:
Its a left over from testing trying to see order doesnt matter and things work as it should, I will removed it sorry for the noise.